<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>379</height>
   </rect>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
    background-color: #1E1E1E;
    color: #E0E0E0;
}
</string>
  </property>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>256</width>
     <height>331</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
    QListWidget {
        background-color: #2b2b2b;
        border: 1px solid #444;
        border-radius: 8px;
        color: #e0e0e0;
        padding: 5px;
        font-family: 'Segoe UI', sans-serif;
        font-size: 14px;
    }

    QListWidget::item {
        background-color: #2b2b2b;
        padding: 10px;
        margin: 5px 0;
        border-bottom: 1px solid #444;
    }

    QListWidget::item:hover {
        background-color: #3d3d3d;
        color: #ffffff;
    }

    QListWidget::item:selected {
        background-color: #6a5acd;
        color: #ffffff;
        border: none;
    }

    QListWidget::item:selected:active {
        background-color: #483d8b;
    }
    </string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>20</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Ville sélectionnée:</string>
   </property>
  </widget>
  <widget class="QLabel" name="desc_label">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>260</y>
     <width>291</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 18px;
font-weight: bold;
margin-bottom: 20px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="textFormat">
    <enum>Qt::PlainText</enum>
   </property>
  </widget>
  <widget class="QLabel" name="temp_label">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>190</y>
     <width>251</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
font-size: 18px;
margin-bottom: 20px;
font-weight: bold;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="textFormat">
    <enum>Qt::PlainText</enum>
   </property>
  </widget>
  <widget class="QLabel" name="image_label">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>150</y>
     <width>161</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">height: 120px;
margin-top: 20px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="etat_n">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>330</y>
     <width>351</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 15px;
font-weight: bold;
color: #FF6F61;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="ville_selec">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>20</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
    QLabel {
        border-radius: 8px;
        border: 1px solid #444;
        padding: 8px;
        font-size: 14px;
        background-color: #2B2B2B;
        color: #FFFFFF;
    }

    QLabel:focus {
        border-color: #1E88E5;
        background-color: #333;
        color: #FFFFFF;
    }
    </string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="boutonoui">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>20</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: #2E2E2E;  /* Fond de la boîte de dialogue */
    border: none;  /* Pas de bordure autour de la boîte */
    padding: 10px;  /* Espacement autour des boutons */
    border-radius: 8px;  /* Coins arrondis */
}

QPushButton {
    background-color: #6A5ACD;  
    color: white; 
    padding: 8px 15px; 
    border: 1px solid #483D8B; 
    border-radius: 6px; 
    font-size: 14px; 
    font-family: 'Segoe UI', sans-serif;  
 
}

QPushButton:hover {
    background-color: #7B68EE;  /* Couleur au survol */
}

QPushButton:pressed {
    background-color: #483D8B;  /* Couleur lors du clic */
}

QPushButton:disabled {
    background-color: #555555;  /* Couleur des boutons désactivés */
    color: #999999;  /* Couleur du texte des boutons désactivés */
}
</string>
   </property>
   <property name="text">
    <string>Oui</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
